book_source/developers_guide/Github-issues.Rmd

```diff
@@ -15,7 +15,7 @@ To learn about GitHub, it is worth taking some time to read through the [FAQ](ht
 
 ### Reporting a bug
 
-1. (For developers) work through debugging (see [debugging](debugging.md) wiki). 
+1. (For developers) work through debugging.
```

Comment:
Is there no debugging link? This seems like a sort of cyclical piece of advice if there's no follow up.